test/libsinsp_e2e/resources/unix_client_server.py

```diff
@@ -27,17 +26,15 @@
   os.remove(NAME)
 
 else:
-  sys.stdout = os.fdopen(sys.stdout.fileno(), 'w', 0)
-
   if os.path.exists(NAME):
     client = socket.socket( socket.AF_UNIX, socket.SOCK_STREAM )
     client.connect(NAME)
 
-    print "STARTED"
+    print(STARTED)
```

Comment:
[nitpick] A variable for two print messages is a bit overkill, don't you think?
```suggestion
    print("STARTED")
```

Reply:
I don't think this is worth the amend :) actually i like it better like this; de gustibus eheh